tests/test_vrs_normalize.py

```diff
@@ -75,7 +75,7 @@
     "type": "Allele",
     "location": {
         "type": "SequenceLocation",
-        "sequence": "refseq:NC_000023.11",
+        "sequence": "ga4gh:SQ.w0WZEvgJF0zf_P4yyTzjjv9oW1z61HHP",
```

Comment:
We should add a regex pattern on the IRI class. I can make a new issue for this

Reply:
> An IRI is a reference to another object. It can be of any form. When we say the sequence slot is dereferenced, it means that instead of an IRI, we have a SequenceReference object. This is true for every property in VRS where we allow for an IRI or object.

Okay, I will update the code + tests to always assume a `SequenceReference`

> We SHOULD NOT assume that an IRI takes a specific form (e.g. a refseq or ga4gh identifier) as we do here.

This was just examples for tests. The `SequenceProxy` class will take the input (regardless of refseq/ga4gh/ensembl etc) to get the corresponding sequence.

Reply:
> When we say the sequence slot is dereferenced, it means that instead of an IRI, we have a SequenceReference object.

@ahwagner thanks for this clarification. The Translator class will need to be updated to work like this (doesn't need to be in this PR). Currently it sets the sequence id (`ga4gh:SQ`, not `ga4gh:SQR` ) as the `location.sequence` value